app/src/main/java/org/openimis/imispolicies/ClientAndroidInterface.java

```diff
@@ -4446,90 +4458,74 @@ public Boolean UploadOfflineFeedbackRenewal(final String ActivityName) {
+                    {
+                        uploadFailed += 1;
+                    } else if (responseCode == 200) {
+                        if (uploadStatus == 1) {
+                            uploadsAccepted += 1;
+                            MoveFile(files[i], 1);
+                            MoveFile(jsonFiles[i], 1);
+                        } else //if(uploadStatus==0)
+                        {
+                            uploadsRejected += 1;
+                            MoveFile(files[i], 2);
+                            MoveFile(jsonFiles[i], 2);
                         }
-
-                        if (resInt != 2) serv = true;
-                        else serv = false;
                     }
 
-                    if (serv == true) {
-                        XMLFile = JSONfiles[i];
-
-                        if (XMLFile.getName().contains("RenPolJSON_")) {
-                            if (resInt == 1) {
-                                MoveFile(XMLFile, 1);
-                                MoveFile(files[i], 1);
-                            } else {
-                                MoveFile(XMLFile, 2);
-                                MoveFile(files[i], 2);
-                            }
-                            result = 1;
-                        } else if (XMLFile.getName().contains("feedbackJSON_")) {
-                            if (resInt == 1) {
-                                MoveFile(XMLFile, 1);
-                                MoveFile(files[i], 1);
-                            } else {
-                                MoveFile(XMLFile, 2);
-                                MoveFile(files[i], 2);
-                            }
-                            result = 1;
-                        } else {
-                            result = 2;
-                        }
+//                    final String status = "Accepted: " + uploadsAccepted + "\n" +
+//                            "Rejected: " + uploadsRejected + "\n" +
+//                            "Failed: " + uploadFailed + "\n";
+//
+//                    ((Activity) mContext).runOnUiThread(new Runnable() {
+//                        @Override
+//                        public void run() {
+//                            ShowDialog(status);
+//                        }
+//                    });
 
-                        ((Activity) mContext).runOnUiThread(new Runnable() {
-                            @Override
-                            public void run() {
-                                ShowDialog(mContext.getResources().getString(R.string.RenewalUploaded));
-                            }
-                        });
-                    } else {
-                        result = -2;
-                    }
                 }
 
-                new Runnable() {
+                if(uploadsAccepted==0 && uploadsRejected==0)
+                {
+                    result = -2;
```

Comment:
Please create a const list with values (-2...) and use that instead of actual value to know what the value means.